src/announcements/views.py

```diff
@@ -194,9 +194,9 @@ def form_valid(self, form):
 
     def get_success_message(self, cleaned_data):
         if self.object.draft:
-            return f"Draft {SiteConfig.get().custom_name_for_announcement} created."
+            return "Draft Announcement created."
         else:
-            return f"New {SiteConfig.get().custom_name_for_announcement} published and broadcast to {SiteConfig.get().custom_name_for_student}s!"
+            return "New Announcement published and broadcast to Students!"
```

Comment:
Leave this one for both announcement and student.  Doesn't need a test.

Reply:
label preserved